ui/org.eclipse.pde.ui.templates/plugin.properties

```diff
@@ -279,7 +279,10 @@ The view is contributed to the workbench by \
 creating a category. The view can be opened by selecting \
 <b>Window</b>, <b>Show View</b> and then <b>Other...</b> \
 on the menu bar. The template demonstrates how data can be exchanged \
-between Java and JavaScript in the browser.
+between Java and JavaScript in the browser.\
```

Comment:
Note, this still doesn't show new text. Please debug that to the point you can see the new text in the wizard.

Reply:
@iloveeclipse - will debug and see what is going on.

Reply:
looks like I was placing the code at a wrong location, fixed it now. @iloveeclipse - can you pls have another look?